File: projects/project_1/project_1/StoreAppModelsLayer/EFModels/Order.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace StoreAppDBContextLayer.EFModels
{
    public partial class Order
    {
        public Guid ItemizedId { get; set; }
        public Guid OrderId { get; set; }
        public int CustomerId { get; set; }
        public int ProductId { get; set; }
        public int StoreId { get; set; }
        public DateTime OrderDate { get; set; }
        public int ProductQuantity { get; set; }
        public int CompletionTime { get; set; }
        public decimal TotalAmount { get; set; }

        public virtual Customer Customer { get; set; }
        public virtual Product Product { get; set; }
        public virtual Location Store { get; set; }
    }
}

File: projects/project_1/project_1/StoreAppUI/wwwroot/order.js
const custId = JSON.parse(sessionStorage.getItem('user')).customerId;
const prodId = JSON.parse(sessionStorage.getItem('product'));
const locId = JSON.parse(sessionStorage.getItem('store'));
const prodPrice = JSON.parse(sessionStorage.getItem('price'));
let date = new Date();


const orderData = { CustomerId: custId, ProductId: prodId, StoreId: locId, OrderDate: date.toJSON(), ProductQuantity: 1, CompletionTime: 1, TotalAmount: prodPrice }

//POST fetch request
fetch(`order/placeorder`, {
    method: 'POST',
    headers: {
        'Accept': 'application/json',
        'Content-Type': 'application/json'
    },
    body: JSON.stringify(orderData)
})
    .then(response => {
        if (!response.ok) {
            console.log('Purchase failed');
            throw new Error(`Network response was not ok (${response.status})`);
        }
        return response.json();
    })
    .then(jsonres => {
        console.log(jsonres);
        const o = document.querySelector('.order');
        o.innerHTML += `<p>Charge: $${prodPrice}</p><br><p>Please confirm</p>`;
    })
    .catch(err => console.log(`There was an error ${err}`));

function Confirm() {
    location.href = "thankyou.html";
}